python/lsst/ts/wep/task/fitDonutRadiusTask.py

```diff
@@ -111,7 +125,9 @@ def run(
         failedFlagArray = []
 
         for stamp in stampSet:
-            xLeft, xRight, donutRadius, flag = self.fit_radius(stamp.stamp_im.image.array)
+            xLeft, xRight, donutRadius, flag = self.fit_radius(
```

Comment:
why are these lines changing? They look essentially the same as before

Reply:
It was black (otherwise line was at >78, PEP8)